### PR TITLE
chore: consolidate consensus nodes into one page

### LINKED ISSUE #556: Consider consolidating consensus and validator pages on docs

### PR DESCRIPTION
**Pull request by jcstein**
## Overview

Replaces #1101 - learned a lesson here.

Resolves #556
Resolves https://github.com/celestiaorg/docs/issues/505

1. [Tested with full node
](https://www.loom.com/share/515bdc2c39374e919cdfe5d2c0760ae0?sid=8ece48d7-49a9-42e1-a28d-d17bc2472852)
2. [tested with self-delegation to valoper address](https://github.com/celestiaorg/docs/pull/1101#issuecomment-1744124046)
3. [tested registering validator](https://github.com/celestiaorg/docs/pull/1101#issuecomment-1744129181)
4. [tested registering evm address for qgb](https://github.com/celestiaorg/docs/pull/1101#issuecomment-1744144333)

## Checklist

- [x] New and updated code has appropriate documentation
- [x] New and updated code has new and/or updated testing
- [x] Required CI checks are passing
- [x] Visual proof for any user facing features like CLI or documentation updates
- [x] Linked issues closed with keywords

### CODE REVIEW

File: docs/developers/celestia-app-vesting.md [hunk 1]
```diff
@@ -382,8 +382,8 @@ celestia-appd tx vesting create-vesting-account $TO_ADDRESS 100000utia 168674805
 Running a consensus full node or validator will prevent you from needing
 to use an RPC.
 
-You can [set up a validator](../../nodes/full-consensus-node) or [full node](../../nodes/validator-node) for the previous portion
-of the tutorial.
+You can [set up a validator or full node](../../nodes/consensus-node)
```
Comment (rootulp) [inline docs/developers/celestia-app-vesting.md:385]: [optional] the previous sentence clarifies that the full node is a consensus full node so proposal to repeat

```suggestion
You can [set up a validator or consensus full node](../../nodes/consensus-node)
```

File: docs/nodes/consensus-node.mdx [hunk 2]
```diff
@@ -143,53 +209,78 @@ cd $HOME<br/>
...
-&nbsp; -C ~/.celestia-app/data/<br/>
+    -C ~/.celestia-app/data/<br/>
 </code></pre>
 
 </TabItem>
-<TabItem value="blockspacerace" label="Blockspace Race">
+<TabItem value="arabica" label="Arabica">
 
-Run the following command to quick-sync from a snapshot for `blockspacerace`:
+Run the following command to quick-sync from a snapshot for `arabica-10`:
```
Comment (rootulp) [inline docs/nodes/consensus-node.mdx:220]: [question] is it possible to interpolate the Arabica chain-id here? Maybe something like: `{constants.arabicaChainId}`?

File: docs/nodes/consensus-node.mdx [hunk 3]
```diff
@@ -143,53 +209,78 @@ cd $HOME<br/>
...
 
-In order to start your validator node, run the following:
+In order to start your full consensus node, run the following:
+
+```sh
+celestia-appd start
+```
+
+Follow the tutorial on
+[setting up Celestia App as a background process with SystemD](./systemd.md).
```
Comment (rootulp) [inline docs/nodes/consensus-node.mdx:245]: [optional] do we want to clarify that this is an optional step?

```suggestion
Optional: If you would like celestia-app to run as a background process, you can follow the [SystemD tutorial](./systemd.md).
```

File: docs/nodes/consensus-node.mdx [hunk 4]
```diff
@@ -143,53 +209,78 @@ cd $HOME<br/>
...
+
+Validator nodes allow you to participate in consensus in the Celestia network.
+
+![validator node](../img/nodes/validator.png)
 
-### Wallet
+#### Validator hardware requirements
+
+The following hardware minimum requirements are recommended for running the
+validator node:
```
Comment (rootulp) [inline docs/nodes/consensus-node.mdx:264]:
```suggestion
The following hardware minimum requirements are recommended for running a
validator node:
```

File: docs/nodes/consensus-node.mdx [hunk 5]
```diff
@@ -484,9 +591,74 @@ txhash: <tx-hash>
...
+based on configuration set in the application.
+
+The options are:
+
+1. `null`
+2. `kv` (default) - the simplest possible indexer backed by key-value storage
+   (defaults to levelDB; see DBBackend)
+   1. when `kv` is chosen `tx.height` and `tx.hash` will always be indexed
+3. `psql` - the indexer services backed by PostgreSQL
+   1. when `kv` or `psql` is chosen, `tx.height` and `tx.hash` will always be indexed
```
Comment (rootulp) [inline docs/nodes/consensus-node.mdx:664]:
```suggestion
   1. when `psql` is chosen, `tx.height` and `tx.hash` will always be indexed
```

**Review by rootulp — APPROVED** [review]
Thanks so much for this! I did a quick scan of https://docs.celestia.org/pr-preview/pr-1107/nodes/consensus-node/ and the rendered output LGTM

File: docs/developers/celestia-app-vesting.md (hunk 1 above)
Comment (jcstein) [reply docs/developers/celestia-app-vesting.md]: matched this PR! https://github.com/celestiaorg/docs/commit/96a13977fe56900518d6423215a041ee44930b71 good flag

File: docs/nodes/consensus-node.mdx (hunk 3 above)
Comment (jcstein) [reply docs/nodes/consensus-node.mdx:245]: absolutely, thank you

File: docs/nodes/consensus-node.mdx (hunk 2 above)
Comment (jcstein) [reply docs/nodes/consensus-node.mdx:220]: yep! using the inline text component:

```mdx
This is a sentence with Arabica's chain-id: <InlineText constant={constants.arabicaChainId} />.
```

Comment (jcstein) [reply docs/nodes/consensus-node.mdx]: 48ad87f67c300b20a5254866d1eaa27c3838e9a1 great catch too, fixed on the other instance as well

File: docs/nodes/consensus-node.mdx (hunk 4 above)
Comment (jcstein) [reply docs/nodes/consensus-node.mdx]: d2cea5356b2a6d07a84b0f778ff47e4a99dc00b8

Comment (jcstein) [reply docs/nodes/consensus-node.mdx]: whoops my b i meant to just commit your suggestion

**Pull request merged**